packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -97,6 +97,22 @@ const Badge = ({
     variant,
     contrast,
   });
+
+  const textSizes = {
+    small: {
+      variant: 'body',
+      size: 'xsmall',
+    },
+    medium: {
+      variant: 'body',
+      size: 'small',
+    },
+    large: {
```

Comment:
medium and large both have same variant and size for Text component? intentional?

Reply:
Yes it same from the start, we only change paddings on medium vs large.